app/javascript/dashboard/helper/localStore.js

```diff
@@ -0,0 +1,23 @@
+export const objectStore = {
```

Comment:
Please use/update the LocalStorage helper available already.

Reply:
https://github.com/chatwoot/chatwoot/blob/develop/app/javascript/dashboard/helper/localStorage.js